tur/rife-ncnn-vulkan-nihui/build.sh

```diff
@@ -0,0 +1,41 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/nihui/rife-ncnn-vulkan
+TERMUX_PKG_DESCRIPTION="RIFE, Real-Time Intermediate Flow Estimation for Video Frame Interpolation implemented with ncnn library"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux-user-repository"
+TERMUX_PKG_VERSION=20221029
+TERMUX_PKG_SRCURL=git+https://github.com/nihui/rife-ncnn-vulkan
+TERMUX_PKG_DEPENDS="libwebp, vulkan-loader"
+TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
+TERMUX_PKG_BUILD_DEPENDS="vulkan-loader-generic"
+TERMUX_PKG_GIT_BRANCH="master"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DUSE_SYSTEM_WEBP=ON"
+TERMUX_PKG_CONFLICTS="rife-ncnn-vulkan-fork"
+
+termux_step_pre_configure () {
+	git submodule update --init --recursive "$TERMUX_PKG_SRCDIR"
+	mv "$TERMUX_PKG_SRCDIR/src/"* "$TERMUX_PKG_SRCDIR/"
+	LDFLAGS+=" -llog -landroid"
+}
+
+termux_step_make_install () {
+	install -Dm700 rife-ncnn-vulkan $TERMUX_PREFIX/bin
+	cp -r models/rife-v2.3 "$TERMUX_PREFIX/bin/rife-v2.3" # default value of model arg
```

Comment:
Installing the models to `$PREFIX/bin` is strange. You can install them to `$PREFIX/opt/$PKG_NAME` and symlink the binary to `$PREFIX/bin`